iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/load/cache/detector/PhiAccrualDetector.java

```diff
@@ -61,17 +65,25 @@ public PhiAccrualDetector(
     this.minHeartbeatStdNs = minHeartbeatStdNs;
     this.codeStartSampleCount = minimalSampleCount;
     this.fallbackDuringColdStart = fallbackDuringColdStart;
+    this.availibilityCache =
```

Comment:
add some comments for why we need cache rather map

Reply:
Done.